package.json

```diff
@@ -2,24 +2,33 @@
   "name": "nimbus-gui",
   "private": true,
   "version": "0.0.0",
-  "type": "module",
   "scripts": {
-    "dev": "vite",
+    "dev": "NODE_ENV=development TAMAGUI_TARGET=web vite",
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview",
```

Comment:
We should add TAMAGUI_TARGET=web in preview and storybook script

Reply:
This should be done in ee25c43, but it turned out we could remove it in all/most cases instead.